src/lib/src/common/markers/marker.state.ts

```diff
@@ -0,0 +1,5 @@
+export class MarkerState {
+  condition: (marker: any) => boolean;
```

Comment:
type any threw me off a bit. may be a short usage description could be helpful here. the respective tests brought some insight but it wasn't immediately clear what restrictions apply to this "type".

Reply:
Added some documentation, but this still assumes some familiarity with our overall concept of "markers". Maybe we should add a section on markers in the project's README, too.